generators/app/templates/plugin.py

```diff
@@ -1,9 +1,15 @@
 # -*- coding: utf-8 -*-
 
 import routing
+import logging
+from resources.lib import utilities
+from resources.lib import kodiLogging
 from xbmcgui import ListItem
 from xbmcplugin import addDirectoryItem, endOfDirectory
 
+
+logger = logging.getLogger(__name__)
```

Comment:
Isn't `__name__` a hidden field exposing the addon name? The only alternative would be to get the addon name via `SCRIPT_ID   = ADDON.getAddonInfo('id')` and use that?